src/sql/workbench/contrib/queryplan2/browser/media/queryPlan2.css

```diff
@@ -3,55 +3,218 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.qp-container .query-plan {
+/* Styling for the a queryplan container in the tab */
+.qps-container .query-plan {
 	width: 100%;
 	height: 500px;
+	display: flex;
+	overflow: hidden;
+}
+
+/* horizontal height resizing sash container that is below a queryplan */
+.qps-container .query-plan-sash {
+	width: 100%;
+	height: 3px;
 	position: relative;
-	border-top: 1px solid;
 }
 
-.qp-container .query-plan .actionbar-container {
-height: 100%;
-width: 30px;
-position: absolute;
-top: 0px;
-right: 0px;
+/*
+The actual sash element constructed by code. Important is used here because the width of the sash is fixed.
+However we always want it to be the width of the container it is resizing.
```

Comment:
Why? Having the sash be the entire width of the container seems a bit strange

Reply:
![image](https://user-images.githubusercontent.com/6816294/151885941-5e06e287-1a2e-445d-a83d-ee49116365c6.png)

The blue bar here is the sash and we need it to take the width of the container so that you can resize it from anywhere on that blue line.

Reply:
Oh, for some reason I assumed this was a vertical sash. It's odd that it requires it but I'm fine leaving it like this if it's broken without that. Not really a huge thing to get hung up on IMO.